minesim.go

```diff
@@ -373,28 +373,25 @@ func main() {
 		minedblocks += m.mined
 		totalstale += m.mined - m.credit
 	}
-	fmt.Printf("seed-arg %d\n", g.seed)
-	fmt.Printf("block-interval-arg %d\n", g.blockinterval)
-	fmt.Printf("stopheight-arg %d\n", g.stopheight)
-	fmt.Printf("total-hashrate-arg %.2f\n",
-		g.totalhash)
-	fmt.Printf("mined-blocks %d\n",
-		minedblocks)
-	fmt.Printf("height %d %.2f%%\n", totalblocks,
-		float64(totalblocks)*100/float64(minedblocks))
-	fmt.Printf("total-simtime %.2f\n",
-		g.currenttime)
-	fmt.Printf("ave-block-time %.2f\n",
+	fmt.Printf("%-20s %-d\n",	"seed-arg", g.seed)
+	fmt.Printf("%-20s %-d\n",	"block-interval-arg", g.blockinterval)
+	fmt.Printf("%-20s %-d\n",	"stopheight-arg", g.stopheight)
```

Comment:
This makes the source more readable, but I have my editor set to automatically run `gofmt`, which removes all the extra spaces. I'd like to maintain this code as `gofmt`-compliant. So let's not make these changes.

Reply:
Done